xcq-extension-core/src/lib.rs

```diff
@@ -1,8 +1,7 @@
 use parity_scale_codec::{Decode, Encode};
-use xcq_extension::Vec;
-use xcq_extension::{DispatchError, Dispatchable};
-use xcq_extension::{ExtensionId, ExtensionIdTy};
+use xcq_extension::extension;
 
+#[extension(0)]
```

Comment:
what is this number for?

Reply:
My fault. Forgot to remove it. I should be carefully read through before creating a PR next time.